easystacks/software.eessi.io/2023.06/eessi-2023.06-eb-4.9.0-2023b.yml

```
@@ -8,3 +8,27 @@ easyconfigs:
   - matplotlib-3.8.2-gfbf-2023b.eb:
       options:
         from-pr: 19552
+  - AOFlagger-3.4.0-foss-2023b.eb:
+      options:
+        from-pr: 19840
+        include-easyblocks-from-pr: 3088
+  - casacore-3.5.0-foss-2023b.eb:
+      options:
+        from-pr: 19840
+        include-easyblocks-from-pr: 3088
+  - IDG-1.2.e0-foss-2023b.eb:
```

It failed due to a typo:
```suggestion
  - IDG-1.2.0-foss-2023b.eb:
```

---

My bad. Thanks for fixing it!